src/Stryker.Core/Stryker.Core/InjectedHelpers/CodeInjection.cs

```diff
@@ -50,16 +52,37 @@ private static string GetRandomNamespace()
             {
                 chars[i] = validChars[random.Next(0, validChars.Length)];
             }
-            return "Stryker" + new string(chars);
+            return StrykerNamespace + new string(chars);
         }
 
         public static IDictionary<string, string> MutantHelpers { get; } = new Dictionary<string, string>();
 
+        public static ObjectCreationExpressionSyntax GetContextClassConstructor() =>
+            SyntaxFactory.ObjectCreationExpression(SyntaxFactory.QualifiedName(
+                SyntaxFactory.IdentifierName(CodeInjection.HelperNamespace),
+                SyntaxFactory.IdentifierName(MutantContextClassName)).WithLeadingTrivia(SyntaxFactory.Space),
+                SyntaxFactory.ArgumentList(),
+                null);
+        
+
+        public static MemberAccessExpressionSyntax GetContextClassAccessExpression(string member) =>
+            SyntaxFactory.MemberAccessExpression(
+                SyntaxKind.SimpleMemberAccessExpression,
+                SyntaxFactory.MemberAccessExpression(SyntaxKind.SimpleMemberAccessExpression,
+                    SyntaxFactory.IdentifierName(CodeInjection.HelperNamespace),
+                    SyntaxFactory.IdentifierName(MutantContextClassName)),
+                SyntaxFactory.IdentifierName(member));
+
+        public static bool IsContextAccessExpression(ExpressionSyntax memberAccess, string member) =>
+            memberAccess is MemberAccessExpressionSyntax
+            {Expression: MemberAccessExpressionSyntax
```

Comment:
This method can use some outlining. It's a bit messy now

Reply:
done, hopes it looks better now. I don't know the rules for pattern matching formatting, I am afraid